domain/infosync/tiflash_manager.go

```diff
@@ -73,6 +74,14 @@ func (m *TiFlashPDPlacementManager) SetPlacementRule(ctx context.Context, rule p
 	if rule.Count == 0 {
 		return m.DeletePlacementRule(ctx, rule.GroupID, rule.ID)
 	}
+	tenant := config.GetGlobalConfig().Tenant
+	if tenant.IsTenant {
+		rule.Constraints = append(rule.Constraints, placement.Constraint{
+			Key:    "tenant-id",
+			Op:     placement.In,
+			Values: []string{strconv.FormatUint(uint64(tenant.TenantId), 10)},
```

Comment:
tiflash will start with the label `tenant-id: xxx`? In this branch, actually, the storage is shared. if not add this constraint, What will be the impact?

Reply:
TiFlash requires a significant changes to support multi-tenant natively. Therefore I decide to not share TiFlash at this time. And we have to make sure tenant matches.

Reply:
got